### DISCUSSION TITLE
Proxy Support

### DISCUSSION THREAD

**Discussion by ptraced** [victornpb/undiscord#524] [3. Ideas]
I've noticed my home ip can delete faster then a vpn so proxy support would be nice.

**Comment by WadeMills** [comment]
Yeah, I totally get what you mean. I've noticed the same thing with my home IP being faster for some tasks compared to using a VPN. Proxy support could definitely be handy in such situations. 
I only use a [mobile proxy](https://proxys.io/en/buy-mobile-proxies) when needed. They've been pretty helpful for specific tasks, but I still rely on my regular setup for most things.
